homeassistant/components/sensor/lacrosse.py

```diff
@@ -0,0 +1,231 @@
+"""
+Support for LaCrosse sensor components.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.lacrosse/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.core import callback
+from homeassistant.components.sensor import (ENTITY_ID_FORMAT, PLATFORM_SCHEMA)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (
+    EVENT_HOMEASSISTANT_STOP, CONF_DEVICE, CONF_FRIENDLY_NAME, CONF_ID,
+    CONF_SENSORS, CONF_TYPE, STATE_UNKNOWN, TEMP_CELSIUS)
+from homeassistant.helpers.entity import Entity, async_generate_entity_id
+from homeassistant.helpers.event import async_track_point_in_utc_time
+from homeassistant.util import dt as dt_util
+
+REQUIREMENTS = ['pylacrosse==0.2.7']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'lacrosse'
+
+CONF_BAUD = 'baud'
+CONF_EXPIRE_AFTER = 'expire_after'
+
+DEFAULT_DEVICE = '/dev/ttyUSB0'
+DEFAULT_BAUD = '57600'
+DEFAULT_EXPIRE_AFTER = 300
+
+TYPES = ['battery', 'humidity', 'temperature']
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_DEVICE, default=DEFAULT_DEVICE): cv.string,
+    vol.Optional(CONF_BAUD, default=DEFAULT_BAUD): cv.string,
```

Comment:
You need something like `vol.Required(CONF_SENSORS): vol.Schema({cv.slug: SENSOR_SCHEMA}),` in the platform validation.